src/core/server/plugins/types.ts

```diff
@@ -154,6 +154,12 @@ export interface PluginManifest {
    */
   readonly requiredPlugins: readonly PluginName[];
 
+  /**
+   * An optional list of component names of the data source OpenSearch plugins that **must be** installed on the cluster
```

Comment:
what exactly does
> data source OpenSearch plugins
mean? From reading the rest of the implementation, it seemed like you could use this for any OpenSearch plugin.

Reply:
Yes that's right!